src/jmclient/blockchaininterface.py

```diff
@@ -457,15 +455,23 @@ def is_address_labeled(self, utxo: dict, walletname: str) -> bool:
     def import_addresses(self, addr_list: Iterable[str], wallet_name: str,
                          restart_cb: Callable[[str], None] = None) -> None:
         requests = []
-        for addr in addr_list:
-            requests.append({
-                "scriptPubKey": {"address": addr},
-                "timestamp": 0,
-                "label": wallet_name,
-                "watchonly": True
-            })
-
-        result = self._rpc('importmulti', [requests, {"rescan": False}])
+        if self.descriptors:
+            for addr in addr_list:
+                requests.append({
+                    "desc": btc.get_address_descriptor(addr),
+                    "timestamp": "now",
```

Comment:
Any specific reason to use `"now"` here instead of `0` like in the `importmulti` path? (afaik they mean the same)

Reply:
`importdescriptors` doesn't have `rescan: False` parameter as `importmulti`, so `now` emulates that here.

Reply:
Oh right, that makes sense.